src/server/search/search_family.cc

```diff
@@ -58,7 +58,7 @@ search::SchemaField::VectorParams ParseVectorParams(CmdArgParser* parser) {
                                   search::VectorSimilarity::COSINE);
     } else if (parser->Check("INITIAL_CAP")) {
       parser->Next<size_t>();
```

Comment:
is line 60 redundant?

Reply:
do we really have testing for this? how can it be that the tests passed when line 61 was under comment?

Reply:
because it isn't used

Reply:
we lack of implementation for a lot of search parameters

Reply:
didn't get at first it's rebase issue

Reply:
just to make sure I understand your comment, we do read an parse the parameter but we did not implement logic how to use it? or we are missing tests here?

Reply:
I think both: the tests I have fixed in next PR https://github.com/dragonflydb/dragonfly/pull/3625 
But the logic is also missing I think